web/sites/default/config/editor.editor.full_html.yml

```diff
@@ -40,7 +40,6 @@ settings:
         -
           name: Links
           items:
-            - Linkit
```

Comment:
It looks like this may have unintentionally removed the "add link" button from our Full HTML text format: 

![image](https://user-images.githubusercontent.com/1520776/235010238-9cd48084-a38b-4d8c-b250-aaa20c765a8e.png)

Reply:
Interestingly this was never enabled when I checked on Test. I've enabled this button.